parm/globus/run_doorman.sh.j2

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+# This script runs on Niagara to interact with the Doorman service
+# Redirect all output to a local file -- it cannot be sent back directly with tty/pty
+script_relpath="$(dirname "${BASH_SOURCE[0]}")"
+log_file="${script_relpath}/run_doorman.log"
+cd "${script_relpath}"
+
+# Initialize the shell (hpss, modules, etc) (this is a Niagara-specific path; parameterize if needed)
+if [[ -f /etc/bashrc ]]; then
+    . /etc/bashrc
+fi
+
+run_doorman(){
+set -ux
+
+rm -f dm.conf
+rm -f places.inc
+rm -f FLIST
+gendel="{{run_directory}}/GENERAL_DELIVERY"
```

Comment:
suggest a better name than 'GENDEL'/'gendel as its not clear what this would be short for

Reply:
That's a good idea.  The GENDEL variable is needed by the doorman service, but I have at least expanded the name of `gendel` to `general_delivery_root`.  I'll pass that suggestion on to George.